ports/stm/boards/stm32f746g_disco/board.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Scott Shawcroft for Adafruit Industries
+ * Copyright (c) 2020 Mark Olsson <mark@markolsson.se>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#include "boards/board.h"
+#include "stm32f7xx_hal.h"
+#include "common-hal/microcontroller/Pin.h"
+
+void board_init(void) {
+    GPIO_InitTypeDef GPIO_InitStructure;
+
+    __HAL_RCC_GPIOK_CLK_ENABLE();
+
+    /*
+     * Turn off the backlight as it is distracting during development.
+     * LCD_BL_CTRL = PK3
+     */
+    GPIO_InitStructure.Pin = GPIO_PIN_3;
```

Comment:
Does this backlight actually turn on when the pin isn't even initialized? Would it be better to set it as a pulldown?

Reply:
Yes, it does. I'll look at setting it as a pulldown though...